tests/scripts/all.sh

```diff
@@ -4824,7 +4824,7 @@ component_test_aes_only_128_bit_keys () {
     scripts/config.py set MBEDTLS_AES_ONLY_128_BIT_KEY_LENGTH
     scripts/config.py unset MBEDTLS_PADLOCK_C
 
-    make CFLAGS='-Werror -Wall -Wextra'
+    make CFLAGS='-O2 -Werror -Wall -Wextra'
```

Comment:
Should we have a comment somewhere near the top "generally build with -O2 so that tests run as quickly as possible?"